amd_openvx_extensions/amd_rpp/source/CropMirrorNormalize.cpp

```diff
@@ -75,7 +76,7 @@ static vx_status VX_CALLBACK refreshCropMirrorNormalize(vx_node node, const vx_r
         data->roi_tensor_ptr[i].xywhROI.roiWidth =data->crop_w[i];
         data->roi_tensor_ptr[i].xywhROI.roiHeight =data->crop_h[i];
     }
-    if(data->layout == 2 || data->layout == 3) // For NFCHW and NFHWC formats
+    if(data->input_layout == 2 || data->input_layout == 3) // For NFCHW and NFHWC formats
```

Comment:
Please change the mean and std_dev updation as offset and multiplier as below

data->mean[index + f] = -(data->mean[n]/  data->std_dev[n]);
data->std_dev[index + f] = (1 / data->std_dev[n]);

Reply:
Done, please check this [commit ](https://github.com/fiona-gladwin/MIVisionX/commit/236ed8a208fbc16750cf061c93af3f8d80de45b3)